commands/proxy.ts

```diff
@@ -0,0 +1,137 @@
+import * as commandParams from "../command-params";
+import { isInteractive } from "../helpers";
+import { EOL } from "os";
+
+const proxyGetCommandName = "proxy|*get";
+const proxySetCommandName = "proxy|set";
+const proxyClearCommandName = "proxy|clear";
+
+export abstract class ProxyCommandBase implements ICommand {
+	public disableAnalytics = true;
```

Comment:
When this option is not set, the commands service tries to track the command usage before executing the command itself. This will fail when the proxy is still not set (for example when callling `<cli name> proxy set`. That's why we disable the tracking here and track the command at the end of the execution method.